hub/core/chunk_engine.py

```diff
@@ -748,6 +761,7 @@ def _update(
         samples: Union[np.ndarray, Sequence[InputSample], InputSample],
         operator: Optional[str] = None,
         update_commit_diff: bool = True,
+        callback: Optional[Callable] = None,
```

Comment:
we should name all these callback args as link_callback or something similar to make it clearer

Reply:
or proper documentation on callback would be suffice.